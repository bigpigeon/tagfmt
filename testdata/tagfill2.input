//tagfmt -f "json=:field"

package main

type Order struct {
    ID  string ``
    Tag string ``
    Fee float32 ``
}
