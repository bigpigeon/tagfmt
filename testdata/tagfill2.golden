//gofmt -f "json=:field"

package main

type Order struct {
	ID  string  `json:"ID"`
	Tag string  `json:"Tag"`
	Fee float32 `json:"Fee"`
}
